src/functional/resources/dateRange/expected/reposense_testrepo-Beta/authorship.json

```diff
@@ -25891,7 +25891,7 @@
         "author": {
           "gitId": "nbriannl"
         },
-        "content": "            + \": Opens a Google Maps view of a person’s address.\\n\""
+        "content": "            + \": Opens a Google Maps view of a personâs address.\\n\""
```

Comment:
doesn't seems to be the optimal solution from the result of the content. Can you look into it further?